packages/server/__tests__/arpa_reporter/server/db/uploads.spec.js

```diff
@@ -42,6 +43,78 @@ describe('db/uploads.js', () => {
             });
         });
     });
+
+    describe('usedForTreasuryExport', () => {
+        before(async () => {
+            await fixtures.seed(knex);
+        });
+
+        after(async () => {
+            await fixtures.clean(knex);
```

Comment:
small improvement that can be handled in a follow-up PR.

I think adding the following code will eliminate the need for lines 70-77, 95-102, 17,19).

```js
describe('db/uploads.js', () => {
    beforeEach(async () => {
        await fixtures.clean(knex)
        await fixtures.seed(knex)
    });
    afterEach(async () => {
        await fixtures.clean(knex)
    });
}
```